.github/workflows/sdk_publish.yaml

```diff
@@ -14,13 +14,44 @@ on:
   workflow_dispatch: {}
 jobs:
   publish:
-    uses: speakeasy-api/sdk-generation-action/.github/workflows/sdk-publish.yaml@v15
-    with:
-      target: my-first-target
-    secrets:
-      github_access_token: ${{ secrets.GITHUB_TOKEN }}
-      java_gpg_passphrase: ${{ secrets.JAVA_GPG_PASSPHRASE }}
-      java_gpg_secret_key: ${{ secrets.JAVA_GPG_SECRET_KEY }}
-      ossrh_password: ${{ secrets.OSSRH_PASSWORD }}
-      ossrh_username: ${{ secrets.OSSRH_USERNAME }}
-      speakeasy_api_key: ${{ secrets.SPEAKEASY_API_KEY }}
+    runs-on: ubuntu-latest
+    env:
+      JRELEASER_GPG_PASSPHRASE: ${{ secrets.SONATYPE_PASSPHRASE }}
+      JRELEASER_GPG_SECRET_KEY: ${{ secrets.SONATYPE_SECRET_KEY }}
+      JRELEASER_DEPLOY_MAVEN_MAVENCENTRAL_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+      JRELEASER_DEPLOY_MAVEN_MAVENCENTRAL_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
+      JRELEASER_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Speakeasy's tooling uses OSSRH, Maven Central's new offering is not included in Speakeasy's tooling. Maven Central's recommended plugins are here: https://central.sonatype.org/publish/publish-portal-gradle/